pycode/memilio-epidata/memilio/epidata/geoModificationGermany.py

```diff
@@ -307,7 +320,153 @@ def get_nuts3_county_id_map(merge_eisenach=True):
     return nuts3_key_dict
 
 
-def merge_df_counties(df, merged_id, separated_ids, sorting=[dd.EngEng['date']], columns=dd.EngEng['date'], method='sum'):
+def create_intermediateregion_level(merge_eisenach=True):
+    """! Creates region information based on county aggregation level which is
+    finer than federal state aggregation and based on mobility.
+    The new regions aggregate a certain level of counties. For more
+    information, see the following references.
+
+    IMPORTANT: This function does not need to be executed. Since the input file 
+    is not publicly available, the results have been copied to defaultDict.
```

Comment:
The problem is with data licenses. This is a nice way to create new county to region assignments if other data is available and it can be adapted easily so that we do not need to rewrite it over and over again.